app/controllers/users_controller.rb

```diff
@@ -0,0 +1,10 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @user = current_user
+    @events = @user.event_atendees
+  end
+
+  def show; end
```

Comment:
for this method you need to list:
- events created by user
- events previoulsy attendented by user
- and upcoming events previously attended by user